packages/web/components/templates/homeFeed/HomeFeedContainer.tsx

```diff
@@ -1229,7 +1229,7 @@ function LibraryItems(props: LibraryItemsProps): JSX.Element {
           data-testid="linkedItemCard"
           id={linkedItem.node.id}
           tabIndex={0}
-          key={linkedItem.node.id}
+          key={linkedItem.node.id + linkedItem.node.image}
```

Comment:
The key was causing the element to not rerender after the article had fully loaded. By appending the image url it will now rerender with the correct image.